src/js/tabs/pid_tuning.js

```diff
@@ -1899,6 +1914,9 @@ TABS.pid_tuning.initialize = function (callback) {
             });
             // enable PID sliders button
             $('a.buttonPidTuningSliders').click(function() {
+                //set Slider PID mode to RPY when re-enabling Sliders
+                FC.TUNING_SLIDERS.slider_pids_mode = 2;
```

Comment:
This part is only available in firmware mode

Reply:
done